dev-libs/aws-c-common/aws-c-common-0.4.57.ebuild

```diff
@@ -12,12 +12,13 @@ SRC_URI="https://github.com/awslabs/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="test"
+IUSE="static-libs test"
 
 RESTRICT="!test? ( test )"
 
 src_configure() {
 	local mycmakeargs=(
+		-DBUILD_SHARED_LIBS=$(usex !static-libs)
```

Comment:
But why? Is there a real [use](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0302) for this?

Reply:
That doesn't mean we should keep static-libs everywhere, though.